### PR TITLE
For checking

### PR DESCRIPTION
**Pull request by EuhenySiniy**

### CODE REVIEW

File: src/main/java/toxml/FormatingToXml.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class FormatingToXml {
+    private static final Logger LOGGER = Logger.getLogger(FormatingToXml.class);
+
+    public void collectionInitialization() {
```
Comment (Taniann) [inline src/main/java/toxml/FormatingToXml.java]: [Method names are typically verbs or verb phrases](https://mate-academy.github.io/style-guides/java/java.html#s5.2.3-method-names)

File: src/main/java/toxml/FormatingToXml.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        Integer value = 10;
+        Set<Integer> set = new HashSet<>(value);
+        List<Set<Integer>> list1 = new ArrayList<>();
+        list1.add(set);
+        Map<List<Set<Integer>>, String> map = new HashMap<>();
+        map.put(list1, "Element");
+        list.add(map);
+    }
+
+    public void conversion(List list) {
```
Comment (Taniann) [inline src/main/java/toxml/FormatingToXml.java]: the same

File: src/main/java/validation/ValidationUtil.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package validation;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ValidationUtil {
+    private final String PHONE_NUMBER = "^\\+\\d{2}\\(\\d{3}\\)\\d{3}-\\d{2}-\\d{2}$";
```
Comment (Taniann) [inline src/main/java/validation/ValidationUtil.java]: also must be static if it is a constant

File: src/main/java/toxml/FormatingToXml.java (hunk 1 above)
Comment (EuhenySiniy) [reply src/main/java/toxml/FormatingToXml.java]: You mean lowerCamelCase?

Comment (EuhenySiniy) [reply src/main/java/toxml/FormatingToXml.java]: You mean lowerCamelCase?

File: src/main/java/toxml/FormatingToXml.java (hunk 2 above)
Comment (EuhenySiniy) [reply src/main/java/toxml/FormatingToXml.java]: the same, or what?

File: src/main/java/toxml/FormatingToXml.java (hunk 1 above)
Comment (Taniann) [reply src/main/java/toxml/FormatingToXml.java]: > You mean lowerCamelCase?

no, I mean verb or verb phrases collectionInitialization -> initializeCollection()

File: src/main/java/toxml/FormatingToXml.java (hunk 2 above)
Comment (Taniann) [reply src/main/java/toxml/FormatingToXml.java]: > the same, or what?

method name should be verb

**Pull request merged**